README.md

```diff
@@ -674,6 +674,7 @@ Any comments, suggestions? [Let us know!](https://github.com/jaywcjlove/awesome-
 * [Grammarly](https://app.grammarly.com/) - Refine your english
 * [iTranslate](http://www.itranslate.com/) - Translate entire website instantly with its built-in browser or with iTranslate Safari extension into over 40 languages. ![Freeware][Freeware Icon]
 * [Ludwig](https://ludwig.guru) - Linguistic search engine that helps you to write better in English.
+* [Bob](https://github.com/ripperhe/Bob) - Translate software, support translate the selected text and screenshot translation with convenient keyboard shortcut (Icon is a little bit out of the ordinary)。[![Open-Source Software][OSS Icon]](https://github.com/ripperhe/Bob) ![Freeware][Freeware Icon] 
```

Comment:
It seems the webpage is Chinese only. It seems inappropriate to add this app to the English version.

Reply:
@viewv